lib/token.js

```diff
@@ -391,7 +391,7 @@ function buildAuthorizeUrl(sdk, oauthParams) {
  *
  *    Optional:
  *      redirectUri: defaults to window.location.href
- *      scopes: defaults to ['openid', 'email']
+ *      scope: defaults to ['openid', 'email']
```

Comment:
Hmm, good question. It looks like we're currently inconsistent:
- We ask for **scope** here
- A **scope** query param is passed to the endpoint
- However, the idToken and accessTokens that we send back use **scopes**

I like the idea of standardizing on **scopes** and deprecating **scope**.